src/esn.inbox.libs/app/services/mailboxes-cache/mailboxes-cache.service.js

```diff
@@ -3,4 +3,6 @@
 angular.module('esn.inbox.libs')
   .factory('inboxMailboxesCache', function() {
     return [];
-  });
+  })
+
+  .factory('InboxMessagesCache', function() { return {}; });
```

Comment:
?

Reply:
It should start with a lowercase `i` instead

Reply:
the `InboxMessagesCache` does nothing, is it intentional?

Reply:
It works like inboxMailboxesCache (but it's in one line and it is a map)

Reply:
@rezk2ll is it clear ?